### ISSUE TITLE
Broken keyboard shortcuts after the latest update

### ISSUE THREAD

**Issue by mlms13** [irrationalistic/atom-tasks#24]
After the update to 1.4.1, Cmd+Enter is creating two lines with checkboxes instead of one.  Also, I don't seem to be able to mark tasks as completed with Cmd+d anymore.

**Comment by irrationalistic** [comment]
Hmm, I'm not seeing that in my local install. Can you try restarting the editor? Does it still apply? What version of atom are you using? Could you try opening the dev tools to see if you are receiving any errors (cmd+i)? Also, try hitting cmd+. to open the key resolver and when you press a key command, see if anything is overriding the cmd-d shortcut?

**Comment by mlms13** [comment]
Strange, I was sure I had already tried restarting Atom, but this time when I restarted, everything sorted itself out.

**Comment by irrationalistic** [comment]
Good to hear! :-)

**Comment by objectkit** [comment]
Hi - apologies for reopening an old issue. Just installed Tasks to Atom 1.16/Mac, but I'm running into a few issues...

- there is no entry for Tasks package in Atom > Packages menu
- there is no entry for Tasks in command palette
- shortcuts aren't working e.g. shift+enter puts newline markers in text files etc

I've disabled/enabled Tasks, and also disabled/enabled keybindings but it hasn't made a difference. I suspect this must be due a package in Atom, but I have no clue.

I would have assumed that creating a new task would have introduced a UI feature of some kind and provided the means to add tasks into a single file hosted or managed by the plugin... or have I got the use case wrong?

Looks like a great package and looking forward to using it - would be great to get some insight into this. Many thanks!
